app/schemas.py

```diff
@@ -396,6 +396,7 @@ class TemplateSchema(BaseTemplateSchema):
     created_by = field_for(models.Template, "created_by", required=True)
     is_precompiled_letter = fields.Method("get_is_precompiled_letter")
     process_type = field_for(models.Template, "process_type")
+    template_category = fields.Nested(TemplateCategorySchema, dump_only=True)
```

Comment:
ah I missed adding the sms_sending_vehicle can we add it in this PR

Reply:
It's a simple type and the `TemplateCategorySchema` references our model. Since you updated it there it picks it up okay in the front end. Here's what it looks like locally in Redis:

```
{
  "id": "d59f9609-b965-4836-bb51-904ee9f3c2b7",
  "name_en": "hmm",
  "name_fr": "hmm",
  "description_en": "hmm",
  "description_fr": "hmm",
  "sms_process_type": "normal",
  "email_process_type": "priority",
  "hidden": false,
  "created_at": "2024-07-08T19:05:13.129712",
  "updated_at": null,
  "sms_sending_vehicle": "long_code"
}
```
template_categories:
```
{
    "id": "b6c42a7e-2a26-4a07-802b-123a5c3198a9",
    "name_en": "Authentication",
    "name_fr": "Authentification",
    "description_en": "Password resets and two factor verification",
    "description_fr": "R\u00e9initialisations de mots de passe et v\u00e9rification \u00e0 deux facteurs",
    "sms_process_type": "priority",
    "email_process_type": "priority",
    "hidden": false,
    "created_at": "2024-07-08T17:03:08.501236",
    "updated_at": "2024-07-08T17:03:08.501236",
    "sms_sending_vehicle": "short_code"
  },
  {
    "id": "977e2a00-f957-4ff0-92f2-ca3286b24786",
    "name_en": "Automatic reply",
    "name_fr": "R\u00e9ponse automatique",
    "description_en": "No-reply and confirmation messages",
    "description_fr": "Messages auxquels il est impossible de r\u00e9pondre et messages de confirmation",
    "sms_process_type": "priority",
    "email_process_type": "priority",
    "hidden": false,
    "created_at": "2024-07-08T17:03:08.501236",
    "updated_at": "2024-07-08T17:03:08.501236",
    "sms_sending_vehicle": "short_code"
  },
....
```